webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code><a>RTCError</a></code> event
+      when the data transport is closed with an error.</li>
```

Comment:
The data channel could also be closed with an error if there aren't enough stream IDs, which occurs above the SCTP level. For example: I call `createDataChannel` 1025 times, but the SCTP handshake (which occurs later) only negotiates 1024 streams. This was something I brought up at a past virtual interim.

So, this situation would need to be differentiated somehow from SCTP protocol errors. I'd propose using errorDetail to differentiate the two, and for SCTP errors, put the Cause Code in a new field (like is done for `httpRequestStatusCode`).